Gordon360/Models/ViewModels/StatesViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Gordon360.Models.CCT;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class StatesViewModel
```

Comment:
Similar to the `CountryViewModel`, what is the purpose of this view model? It has exactly the same structure as the `States` db model so why not just use the `States` class directly?

Reply:
did not know that I could use them from the auto generated file when I was first looking into this, and that format was copied onto countries